### ISSUE TITLE
Style Main Page subsections

### ISSUE THREAD

**Issue by cl8n** [ppy/osu-wiki#2728]
idk why we kept these group/subsection things in parenthesis, they look terrible and get split across lines

**Comment by Circleen** [comment]
#2731 Feel free to edit anything here.

**Comment by sr229** [comment]
I'm more in favor of #2730 than #2731.

**Comment by TicClick** [comment]
I guess this is not resolved yet, and the main page blocks need further improvements, although they definitely look better without braces ([context](https://user-images.githubusercontent.com/4686101/119195193-841eea00-ba84-11eb-957e-4f2f409d6598.png))

**Comment by TicClick** [comment]
the main page rewrite for #2606 is in progress, marking this as `blocked`

**Comment by cl8n** [comment]
been a long time but I think I was trying to say inline styling for these sections isn't the way to go. parenthesis or not, the sub-section-titles have no significance compared to the articles within, and titles wrap unnecessarily, and extra separator dots appear at end of line sometimes, ...

as far as wiki is concerned these should probably just be lists contained in a styling block, and web can take care of how it looks. I was looking into rewriting the main page renderer anyway cuz the setup is really odd web-side. syntax would be something like this for each panel. headers and lists would have custom styling under `Panel` instead of the classes applied via HTML currently

?

```markdown
{{{ Panel

## People

*some description from #5506*

{{{ Panel Section

### [The Team](/wiki/People/The_Team)

- [Developers](/wiki/People/The_Team/Developers)
- [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team)
- ...

}}}

...

}}}
```

**Comment by TicClick** [comment]
thought it was wiki only and the changes would be more drastic, which is tough to do at the same time as updating the page

**Comment by TicClick** [comment]
rewrite done. reposting important stuff from the pull:

- https://github.com/ppy/osu-wiki/pull/5506#issuecomment-851706553 LiquidPL's thoughts
- https://github.com/ppy/osu-wiki/pull/5506#issuecomment-855259985 GPR's design experiment
- https://github.com/ppy/osu-wiki/pull/5506#issuecomment-888869508 cl8n's thoughts

also, infobox example: #5440

**Comment by Walavouchey** [comment]
is this still tracking anything?

**Comment by TicClick** [comment]
yes, it still looks poor and the subsections are not styled at all (this needs *someone* to experiment with the design and find *something* similar to https://github.com/ppy/osu-wiki/pull/5506#issuecomment-855259985 which didn't happen)

**Comment by Walavouchey** [comment]
the preview in the comment you linked looks exactly like how the main page looks now, so i'm not sure on what you mean there. but i agree that the uh... *design* of the main page is quite dull
